reactDashboard/src/components/DataCharts/DataCharts.css

```diff
@@ -1,13 +1,27 @@
 .chart-border {
   box-shadow: 10px 10px 5px #aaaaaa;
-  padding: 10px 50px 80px 50px;
+  padding: 10px 50px 80px;
+  margin-left: auto;
+  margin-right: auto;
   display: flex;
   flex-direction: column;
   justify-content: center;
-  background-color: #ffffff;
+  background-color: white;
   position: relative;
 }
 
+.darkmode .chart-border {
+  background-color: rgb(64, 64, 64);
+  box-shadow: none;
+}
+
+.chart-border .container,
+.chart-border .container-lg,
+.chart-border .container-md,
+.chart-border .container-sm {
```

Comment:
Not sure of the reason for this override, but don't you also want to apply it to xs ?

Reply:
✅

Reply:
Not sure what :white_check_mark: means here

Reply:
added xs and xl and removed media query - it finally behaves as was intended and chart stays full width on re-size.  Thanks.